core/src/main/java/org/infinispan/factories/impl/BasicComponentRegistryImpl.java

```diff
@@ -67,6 +71,14 @@ public <T> ComponentRef<T> lazyGetComponent(Class<T> componentType) {
       return getComponent0(componentType.getName(), componentType, false);
    }
 
+   @Override
+   public void blameInitialization() {
+      if (tracker != null && log.isTraceEnabled()) {
+         log.tracef("Initialization blame of %s:%n%s", scope, tracker.dump());
```

Comment:
I probably wouldn't say "blame" but maybe something like "Component initialization performance:" or something like that.